tests/drivers/uart/uart_mix_fifo_poll/testcase.yaml

```diff
@@ -0,0 +1,56 @@
+tests:
```

Comment:
This should probably have a README explaining how it's supposed to work.  Does it not require an external loopback connection to short RX/TX CTS/RTS?